metadata.yaml

```diff
@@ -54,7 +54,6 @@ spec:
       - name: project_id
         description: Google Cloud Project ID
         varType: string
-        defaultValue: null
```

Comment:
What's driving this change? I'm not sure if it's needed for the console deployment.

Reply:
I haven't touched it manually. Probably it was automatically generated by the docs.

Reply:
Likely related to this fix: https://github.com/GoogleCloudPlatform/cloud-foundation-toolkit/pull/2048